### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by benja-wu**

### CODE REVIEW

**Review by megaeasex — APPROVED** [review]
[TASK:easegress-pr-test SUCCESS]megaease/easegress Pull Request 162 Deploy Test Success

File: README.md [hunk 1]
```diff
@@ -15,6 +15,7 @@
     - [More Filters](#more-filters)
   - [Documentation](#documentation)
   - [Roadmap](#roadmap)
+  - [Slack](#slack)
```
Comment (haoel) [inline README.md:18]:
```suggestion
  - [Community](#community)
```

File: README.md [hunk 2]
```diff
@@ -298,6 +299,11 @@ See [reference](./doc/reference.md) and [developer guide](./doc/developer-guide.
 
 See [Easegress Roadmap](./doc/Roadmap.md) for details.
 
+## Slack
+
+Welcome to our [slack channel](https://openmegaease.slack.com/archives/C028Z6SUCU9) for products and details discussion.
+
+
```
Comment (haoel) [inline README.md:306]:
```suggestion
## Community

- [Slack Channel](https://openmegaease.slack.com/archives/C028Z6SUCU9) for requirement, issue and development.
- [MegaEase on Twitter](https://twitter.com/megaease) 


```

Comment (benja-wu) [reply README.md]: updated

**Review by megaeasex — APPROVED** [review]
[TASK:easegress-pr-test SUCCESS]megaease/easegress Pull Request 162 Deploy Test Success

**Review by localvar — APPROVED** [review]

**Review by haoel — APPROVED** [review]

**Pull request merged**